test/integration/PoolUpdatesWithHints.js

```diff
@@ -121,56 +135,100 @@ describe("PoolUpdatesWithHints", () => {
-        assert.equal(await transcoderAtPoolPos(size - 1), transcoders[size - 1].address)
+        const txResHint = await (
+            await bondingManager
+                .connect(transcoders[size - 1])
+                .rewardWithHint(
+                    transcoders[size - 2].address,
+                    ethers.constants.AddressZero
+                )
+        ).wait()
+        assert.equal(
+            await transcoderAtPoolPos(size - 1),
+            transcoders[size - 1].address
+        )
 
         // Gas cost of rewardWithHint() should be less than gas cost of reward()
-        assert.isBelow(txResHint.cumulativeGasUsed, txResNoHint.cumulativeGasUsed)
+        assert.isBelow(
+            txResHint.cumulativeGasUsed,
+            txResNoHint.cumulativeGasUsed
+        )
     })
 
     it("new transcoder joins the pool", async () => {
         const size = transcoders.length
         await approve(transcoders[size - 2], 1)
-        await bondingManager.connect(transcoders[size - 2]).bond(1, transcoders[size - 2].address)
+        await bondingManager
+            .connect(transcoders[size - 2])
+            .bond(1, transcoders[size - 2].address)
         await approve(transcoders[size - 1], 1)
-        await bondingManager.connect(transcoders[size - 1]).bond(1, transcoders[size - 1].address)
+        await bondingManager
+            .connect(transcoders[size - 1])
+            .bond(1, transcoders[size - 1].address)
 
         // Not enough stake to join pool
         await approve(newTranscoder, 2)
-        await bondingManager.connect(newTranscoder).bond(2, newTranscoder.address)
+        await bondingManager
+            .connect(newTranscoder)
+            .bond(2, newTranscoder.address)
 
         // After this tx, the new transcoder should have enough stake to join pool
         await bondingManager.connect(transcoders[size - 1]).unbond(1)
         const dr = (await roundsManager.currentRound()).add(1)
         const testSnapshotId = await rpc.snapshot()
 
-
         // Pool ordering (descending)
         // (transcoders[size - 4], 4) -> (transcoders[size - 2], 3) -> (transcoders[size - 3], 3) -> (transcoders[size - 1], 2)
 
         // Get gas cost of transcoder()
         let tx = await bondingManager.connect(newTranscoder).transcoder(0, 0)
         const txResNoHint = await tx.wait()
+        await expect(tx.hash)
+            .to.emit(bondingManager, "TranscoderDeactivated")
+            .withArgs(transcoders[size - 1].address, dr)
 
         assert.equal(await transcoderAtPoolPos(size - 1), newTranscoder.address)
-        expect(tx).to.emit(bondingManager, "TranscoderDeactivated").withArgs(transcoders[size - 1].address, dr)
+        await expect(tx)
+            .to.emit(bondingManager, "TranscoderDeactivated")
+            .withArgs(transcoders[size - 1].address, dr)
```

Comment:
Can actually remove this
Line 197 does the same thing.

Reply:
Fixed in 6ac1cb5